docs/manual/input.md

```diff
@@ -607,15 +607,15 @@ The rule needs to be applied to an existing 2D/3D variables with or without time
 
 It is possible to include data in the YAML file that originates from another file. At the moment this is only applicable to another YAML file. This can be useful for storing large classification_rule tables in a separate file (for a better overview of the work file), but this functionality is not limited to that specific rule.
 
-Here is the original rule: 
+Here is the original rule:
 ```
 #EXAMPLE  : Original
 # This is a simplified example, only top layer of flow velocity and chloride was used and year statistics
 
   - classification_rule:
       name: classification for aquatic plants
       description: classification for aquatic plants based on water depth, flow velocity and chloride.
-      criteria_table: 
+      criteria_table:
         - ["output", "MIN_water_depth_mNAP", "MAX_flow_velocity", "MAX_chloride"]
         - [     1  ,               "<0.10" ,                "-" ,            "-"] # too dry
         - [     2  ,                ">4.0" ,                "-" ,            "-"] # too deep
```

Comment:
this file should also have documentation on the new rule, correct?

Reply:
forgot that, will add